### PR TITLE
Add: new meeting notes April 23

### LINKED ISSUE #140: 20200423 - meeting
Just an hour hanging out and talking about DAT stuff.

:point_right: **Post a comment in this github issue with any topics you'd like to cover this week**

:alarm_clock: Time: 23 Apr 2020, 18:00 GMT

:link: https://zoom.us/j/181036238 (Native)
:link: https://zoom.us/wc/join/181036238?pwd= (Web)

| Timezone      |  Date/Time           |
| ------------- |:-------------:|
| Buenos Aires  | 23 Apr, 3 PM |
| Toronto       | 23 Apr, 2 PM      |
| New York      | 23 Apr, 2 PM      |
| Los Angeles   | 23 Apr, 11 AM      |
| Berlin        | 23 Apr, 8 PM      |

Or in your local time: https://www.timeanddate.com/worldclock/fixedtime.html?msg=Dat+Comm-Comm+Meeting+2020-04-23&iso=20200423T18&p1=1440&ah=1

### PR DESCRIPTION
**Pull request by dpaez**
New meeting notes for todays comm-comm meeting. Feel free to add/adjust anything.

Thanks to everyone who joined 👏 

Closes #140

### CODE REVIEW

File: meeting-notes/49-23Apr2020.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+* Diego (@dpaez)
+* mauve (@rangermauve)
+* Kevin Faaborg (@zootela)
+* Peg
+
+## Meeting Notes
+
+### Intro
+
+- mauve: working with datdot and ... .Release a couple of projects recently (https://github.com/RangerMauve/p2plex/). Multiplex peers streams. Chat, apps, and different things going over a single connection. More easily to dedup connections. Similar idea to libp2p: combine the discovery and encryption and multiplex. Initially done for the datdot project. [dat gossip](https://github.com/RangerMauve/dat-gossip) like a PoC of gossiping keys, eg chat app sharing hypercores, you can use this gossip module for share these hypercores and easily discover data.
```
Comment (RangerMauve) [inline meeting-notes/49-23Apr2020.md:16]:
```suggestion
- mauve: working with datdot and ... .Release a couple of projects recently (https://github.com/RangerMauve/p2plex/). Multiplex peers streams. real-time communicaton apps, and different things going over a single connection. More easily to dedup connections. Similar idea to libp2p: combine the hyperswarm for peer discovery and noise for encryption / authentication and multiplex for sending multiple streams of data at once. Made for the datdot project. [dat gossip](https://github.com/RangerMauve/dat-gossip) like a PoC of gossiping keys, eg chat app sharing hypercores, you can use this gossip module for share these hypercore keys and easily discover data.
```

File: meeting-notes/49-23Apr2020.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+mauve: one vector clock or more like a profile thing?
+peg: profile (multiple vectors?)
+mauve: was looking around for different approaches.
+
+peg: https://gitlab.com/coboxcoop/kappa-drive/-/blob/master/index.js#L502
+https://gitlab.com/coboxcoop/kappa-drive/-/blob/master/index.js#L383
+
+mauve: why store it in kappa instead of metadata?
+peg: it is in the feed and the feed is aggregated using kappa core.
+mauve: trade off?
```
Comment (RangerMauve) [inline meeting-notes/49-23Apr2020.md:45]:
```suggestion
mauve: is the tradeoff storing a bit of stuff in leveldb for improved reads?
```

File: meeting-notes/49-23Apr2020.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+peg: it was the way we see it.
+
+### static site gens
+
+kevin: every website starts with an index.html and that's it. You can upload and even edited from the browser. That's the feeling that I'm looking for.
+This can work on beaker and everywhere actually.
```
Comment (RangerMauve) [inline meeting-notes/49-23Apr2020.md:51]:
```suggestion
This can work on beaker and everywhere actually, like [zeronet](https://zeronet.io/)
```

**Comment by RangerMauve** [comment]
Added some slight details. 💜

**Comment by dpaez** [comment]
@RangerMauve thank you mauve you are the best!

**Pull request merged**